csharp/extractor/Semmle.Extraction.CSharp/Entities/UserOperator.cs

```diff
@@ -166,6 +166,13 @@ public static bool OperatorSymbol(string methodName, out string operatorName)
                     operatorName = @checked;
                     break;
                 default:
+                    var match = Regex.Match(methodName, "^op_Checked(.*)$");
+                    if (match.Success)
+                    {
+                        OperatorSymbol("op_" + match.Groups[1], out var uncheckedName);
+                        operatorName = "checked " + uncheckedName;
+                        break;
+                    }
```

Comment:
Do we need this in the `default` branch? Why wasn't this added in the `var @checked when Regex.IsMatch(@checked, "^op_Checked.*"):` case? Or maybe the `var @checked when ...` could be removed now.

Reply:
But isn't the below redundant?
```
case var @checked when Regex.IsMatch(@checked, "^op_Checked.*"):
                    operatorName = @checked;
                    break;
```

Reply:
Ah 🤦 I thought I had removed it; is removed now.